docs/protocol_overview.md

```diff
@@ -0,0 +1,245 @@
+12. In Vetomint, validators can veto the current block proposer, but still the round will progress (changing the block proposer) if all the honest validators either vote or veto.
+
+## Simple and Light Node Operation
+
+Simperby strongly encourages each member to run their own node,
+in order to ensure that the network is decentralized and distributed (and so truly self-hosted). In other words, running a Simperby chain is not about using AWS servers by a few people, but using their own laptops to physically run the chain. To realize that, it is important to **keep the node operation as simple and lightweight as possible**.
+
+### Rarely-online nodes
+
+One of the most important condition for a 'simple' and 'lightweight' node operations is how often the node needs to be online.
+If we want to make the validators run nodes on their personal laptops, for example, it is not realistic to assume that they will be online 24/7.
+
+Since Simperby's BFT consensus assumes an partially-synchronous network, rarely-online nodes can be trivially handled because it's no more than one typical case of an asynchrony, if
+
+1. the consensus round is (or has grown to be) long enough to cover all appearances of the nodes.
+2. at least one node is online where there is a network broadcast
+
+The condition 1 is actually turned out to be a tough challenge in the later sections, so keep in mind it.
+
+### On-demand block production
+
+Rarely-online nodes means rarely-progressed blockchain.
+(note that in any BFT algorithm, at least 2/3 of nodes must have participated in the consensus process to produce a block)
+
+This will inevitably compromise the latency or throughput of the blockchain, but it really doesn't matter because
+**Simperby is NOT a general contract platform for serving public users,**
+**but a governance platform for a permissioned set of members**.
+
+Simperby blockchain includes only governance-approved state transitions (there are very [few exceptions](#consensus-leader)).
+Every governance agenda must be approved by a majority vote of the members so it will take a time anyway (days or even weeks).
+Considering that, naturally, Simperby's performance will be bounded by the governance process, not the consensus slowed by lightweight node operation.
+
+### Multichain DAO
+
+One of the notable use cases of Simperby is that it can establish a **muli-chain DAO**. (See [this](./multichain_dao.md) for more details)
+it's super-easy to expand the organization to other existing chains (so called 'colony chains') using
+the light client of the Simperby consensus.
+Since the light client is uploaded as a contract to other chains,
+any block progress of the Simperby consensus will require an additional
+transaction to update and store the Merkle root of it.
+This might not be cheap, because **the organization will have to pay for the gas cost for every colony chains, for every Simperby block**.
```

Comment:
```suggestion
This might not be cheap, because **the organization will have to pay for the gas cost for every colony chain, for every Simperby block**.
```